client/src/components/Home.js

```diff
@@ -2,10 +2,13 @@ import React from "react";
 import LandingPage from "./LandingPage";
 import { getLoggedInUserData } from "../utils/storage";
 import MatchingApplicantsAndOppotunities from "./MatchingApplicantsAndOppotunities";
+import ApplicantsList from "../components/Applicant/ApplicantsList";
 
 export default () =>
-  getLoggedInUserData() ? (
+  getLoggedInUserData() && getLoggedInUserData().user.role === "applicant" ? (
     <MatchingApplicantsAndOppotunities />
+  ) : getLoggedInUserData() && getLoggedInUserData().user.role === "company" ? (
```

Comment:
It is not good to use nested ternary condition instead you can use a function, like this.
```
const renderComponent = ()  => {
  if (getLoggedInUserData() && getLoggedInUserData().user.role === "applicant") {
    return <MatchingApplicantsAndOppotunities />;
  }
  if (getLoggedInUserData() && getLoggedInUserData().user.role === "company") {
    return <ApplicantsList />;
  }
  return  <LandingPage />;
}

export default () => (renderComponent());```